portality/templates/application_form/js/_form_js.html

```diff
@@ -29,6 +29,9 @@
             {% if form_diff %}
                 formDiff: {{ form_diff|tojson }},
             {% endif %}
+            {% if obj.admin.current_journal is not none and (not current_journal or not current_journal.is_in_doaj()) %}
+                withdrawn: True
```

Comment:
`True` is the python true, for javascript it needs to be `true`

Reply:
Fixed. @Steven-Eardley I think we may update that on the test server now.